client/src/component/itemList.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import List from '@material-ui/core/List';
+import Button from '@material-ui/core/Button';
+import { v4 as uuidv4 } from 'uuid';
+import Item from './item';
+
+
+const ItemList = ({ticketItems}) => {
+    const [itemList, setItemList] = useState([]);
+
+    const removeItem = (index) =>{
+        const newItemList = itemList.filter(item => item.id !== index);
+        setItemList(newItemList);
+    }
+
+    useEffect(() => {ticketItems(itemList)}, [itemList])
```

Comment:
I was originally confused about what this was doing. I think you just also need to name the function `setTicketItems`.